src/components/Nav.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { getTopics } from '../utils/api';
+
+const Nav = () => {
+	const [topics, setTopics] = useState([]);
+	useEffect(() => {
+		getTopics()
+			.then((topicsFromApi) => {
+				setTopics(topicsFromApi);
+			})
+			.catch((err) => console.log(err));
+	}, []);
+
+	if (!topics) {
+		return <nav>Loading...</nav>;
+	} else {
+		return (
+			<nav>
+				<li key='Home'>
+					<NavLink to='/'>Home</NavLink>
+				</li>
+
+				{topics.map((topic) => {
+					return (
+						<li key={topic.slug}>
+							<NavLink to={`/articles/${topic.slug}`}>{topic.slug}</NavLink>
```

Comment:
We don't currently have an articles/:topic endpoint currently, which means when navigating through the different topics in the Nav, react is throwing a warning that we have not accommodated for this in the routing.